op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```diff
@@ -0,0 +1,217 @@
+# Runbook: Incident Response for Faultproof Withdrawals
+- [Runbook](#runbook)
+    - [Overview](#overview)
+    - [Incident Management](#incident-management)
+    - [General Metrics and Alerts Descriptions](#general-metrics-and-alerts-descriptions)
+    - [General Incident Response Guidelines](#general-incident-response-guidelines)
+    - [Conclusion](#conclusion)
+
+## Overview
+
+This document serves as a guide for incident response based on key metrics related to faultproof withdrawals. It describes the alerts triggered by specific conditions in the system and provides guidelines on how to handle these alerts.
+
+FaultproofWithdrawal will monitor one L2 chain, let's say op-chain, and one L1 chain, let's say mainnet.
+On L2, it will make use of [L2ToL1MessagePasser](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L2/L2CrossDomainMessenger.sol).
+On L1, it will make use of [OptimismPortal2](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L1/OptimismPortal2.sol) and [FaultDisputeGame](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol).
+
+The monitor is driven by the event [WithdrawalProvenExtension1(bytes32 indexed withdrawalHash, address indexed proofSubmitter)](https://github.com/ethereum-optimism/optimism/blob/dd2b21ce786f4c1b722bda270348597182153c8e/packages/contracts-bedrock/src/L1/OptimismPortal2.sol#L144C5-L144C102). Every time an event is emitted, the monitor will check if this withdrawal is legitimate or a forgery attempt.
+
+## ⚠️ Disclaimer: work in progress!
```

Comment:
nit, would suggest using markdown callouts for disclaimers like this: https://github.com/orgs/community/discussions/16925